relevanceai/api/endpoints/deployables/dashboard.py

```diff
@@ -0,0 +1,138 @@
+from abc import ABC, abstractmethod
+from typing import Collection
+
+from relevanceai.api.endpoints.datasets.datasets import DatasetsClient
+from relevanceai.base import _Base
+from .deployables import Deployable
+
+
+class Dashboard(ABC, _Base):
+    def __init__(
+        self, project: str, api_key: str, deployable_id: str, application: str
+    ):
+        deployables = Deployable(project, api_key)
+        if deployable_id not in deployables.list():
+            raise ValueError(f"No deployable with ID {deployable_id}")
+        else:
+            configuration = deployables.get(deployable_id)["configuration"]
+            if not application == configuration["type"]:
+                raise ValueError(f"{deployable_id} is not a {application} application")
+            else:
+                self.vector_field = configuration["vector_field"]
+
+        self.project = project
+        self.api_key = api_key
+        self.deployable_id = deployable_id
+        self.shareable_id = None
+
+    @property
+    @abstractmethod
+    def deployable_url(self):
+        pass
+
+    @property
+    @abstractmethod
+    def shareable_url(self):
+        pass
+
+    @classmethod
+    @abstractmethod
+    def create_application(cls, project: str, api_key: str):
+        pass
+
+
+class Clusters(Dashboard):
```

Comment:
This would be like `ClusterDashboard` ?

Reply:
I haven't settled on concrete names yet, so we can finalize later. My motivation for naming this "Clusters" is just because on the Relevance AI dashboard the name of the application (Under "Apps") is Clusters.